script/output/mainnet/M2_mainnet_upgrade.output.json

```diff
@@ -0,0 +1,35 @@
+{
+  "addresses": {
+    "avsDirectory": "0x135DDa560e946695d6f155dACaFC6f1F25C1F5AF",
+    "avsDirectoryImplementation": "0xdAbdB3Cd346B7D5F5779b0B614EdE1CC9DcBA5b7",
+    "baseStrategyImplementation": "0xdfdA04f980bE6A64E3607c95Ca26012Ab9aA46d3",
```

Comment:
This is the old strategy implementation - are we going to deploy + upgrade to the new one?

Reply:
because there have been zero logical or storage changes, I believe it would be unnecessary to do so.
I acknowledge that the DelayedWithdrawalRouter is in a similar situation, but I let that one go 🤷